src/programmers/helpers/ITagFormatValidator.ts

```diff
@@ -0,0 +1,9 @@
+import { IMetadataTag } from "../../metadata/IMetadataTag";
+
+export namespace ITagFormatValidator {
```

Comment:
Not sure if this is placed right here, just saw that the folder holds some functionality that is being re-used in `internals`.